specs/AppNotifications/AppNotificationContentSpec/AppNotificationBuilder-spec.md

```diff
@@ -0,0 +1,1170 @@
+  .GetXml();
+```
+
+XML output:
+
+```c#
+<toast>
+  <visual>
+    <binding template="ToastGeneric">
+      <text>Above is a Hero image</text>
+      <image src="https://unsplash.it/360/180?image=1043"
+                placement="hero" />
+    </binding>
+  </visual>
+</toast>
+```
+
+## Cropped AppLogoOverride
+
+For both Inline and AppLogoOverride, developers can set how the image will be cropped.
+
+![Cropped AppLogoOverride Image Example](CroppedAppLogoOverrideExample.png)
+
+AppNotificationContent code:
+
+```c#
+AppNotificationContent()
+    .AddText(L"To the left is a cropped AppLogoOverride")
+    .SetAppLogoOverride(winrt::Windows::Foundation::Uri(LR"(https://unsplash.it/64?image=669)"), ImageCrop::Circle)
+    .Show();
+```
+
+XML output:
+
+```c#
+<toast>
+  <visual>
+    <binding template="ToastGeneric">
+      <text>To the left is a cropped AppLogoOverride</text>
+      <image src="https://unsplash.it/64?image=669"
+                hint-crop="circle"
+                placement="appLogoOverride" />
+    </binding>
+  </visual>
+</toast>
+```
+
+# Audio
+
+The Audio component allows the developer to define a custom audio to play when an AppNotification is
+displayed. The audio file can be defined by string value pointing to an app asset or one of the
+[ms-winsoundevents](https://docs.microsoft.com/en-us/uwp/schemas/tiles/toastschema/element-audio).
+
+**WinAppSDK 1.2 \<audio\> Schema**:
+
+```c#
+<audio src? = string
+       loop? = boolean
+       silent? = boolean />
+```
+
+These attributes are abstracted with the Audio component APIs:
+
+-   SetAudio(Windows.Foundation.Uri audioUri)
+-   SetAudio(MSWinSoundEvent winSoundEvent)
+-   SetAudio(Windows.Foundation.Uri audioUri, AppNotificationDuration duration)
+-   SetAudio(MSWinSoundEvent winSoundEvent, AppNotificationDuration duration)
+-   MuteAudio()
+
+Below are some example usages:
+
+```c#
+AppNotificationContent()
+      .SetAudio(MSWinSoundEvent::Reminder)
+    GetXml();
+```
+
+XML output:
+
+```c#
+<toast>
+  <visual>
+    <binding template="ToastGeneric">
+    </binding>
+  </visual>
+  <audio src="ms-winsoundevent:Notification.Reminder" />
+</toast>
+```
+
+Developers can define if the audio should be looped using SetAudio(Uri, AppNotificationDuration),
+which can be short or long.
+
+```c#
+AppNotificationContent()
+    .AddText(L"Content")
+    .SetAudio(MSWinSoundEvent::Alarm, AppNotificationDuration::Long)
+    .GetXml();
+```
+
+XML output:
+
+```c#
+<toast duration="long">
+  <visual>
+    <binding template="ToastGeneric">
+        <text>Above is a Hero image</text>
+    </binding>
+  </visual>
+  <audio src="ms-winsoundevent:Notification.Looping.Alarm" loop="true" />
+</toast>
+```
+
+An AppNotification that does not want to play any audio including the default, can use MuteAudio to
+mute the audio.
+
+```c#
+AppNotificationContent()
+    .AddText(L"Content")
+    .MuteAudio()
+    .Show();
```

Comment:
I thought we were not exposing Show() anymore. 
Suggest that we do the following:
1) Get compilable code against the idl
2) Run these little snippets here to make sure that they compile against the latest idl to prevent these kind of errors.

Reply:
Going to double check all samples before resolving this comment.